mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -362,10 +360,12 @@ static TransformingForOp
 createGlobalLoadLoop(PatternRewriter &b, Location loc, GpuAllocOp loadBuffer,
                      Value wrappedMatrix, ArrayAttr vectorMap,
                      int64_t dataPerThread, int64_t vectorLen, Value bid,
```

Comment:
Instead of bid here,  lets pass in the g_block val, m_block val and n_block val if this function is being kept.
I still think we can replace the whole function with a threadwise_read_jnto.